### DISCUSSION TITLE
Clock Generator Calibration sketch (Broken Link in Wiki)

### DISCUSSION THREAD

**Discussion by halfmanbear** [sanni/cartreader#240] [Need help]
The following Dropbox link appears to be broken.

https://www.dropbox.com/s/uyhpnng9zgfl2ja/clkcal.zip?dl=1

Cheers

**Comment by sanni** [comment]
[clkcal_sketch.zip](https://github.com/sanni/cartreader/files/7097399/clkcal_sketch.zip)
